rust-toolchain.toml

```diff
@@ -7,7 +7,7 @@
 #   supported and used in CI.
 
 [toolchain]
-channel = "1.60.0"
```

Comment:
with https://github.com/mozilla/uniffi-rs/pull/1314 landed, we'll be able to pick up 1.63 soon too!

Reply:
aaah I realize that's exactly what you said on the PR message